spec/features/state_file/complete_intake_spec.rb

```diff
@@ -738,5 +768,303 @@
+    end
+
+    it "handles property tax both flow - eligible homeowner - when low income but meets exemption", required_schema: "nj" do
+
+      advance_to_start_of_intake("Minimal") # low income
+      advance_county_and_municipality
+      advance_disabled_exemption(true) # disabled exemption
+      advance_veterans_exemption
+      advance_medical_expenses
+
+      # household rent/own page
+      choose I18n.t('state_file.questions.nj_household_rent_own.edit.both')
+      click_on I18n.t("general.continue")
+
+      # homeowner eligibility page
+      expect(page).to have_text I18n.t("state_file.questions.nj_homeowner_eligibility.edit.title", filing_year: filing_year)
+      check I18n.t('state_file.questions.nj_homeowner_eligibility.edit.homeowner_home_subject_to_property_taxes')
+      click_on I18n.t("general.continue")
+
+      # skips property tax paid page
+      # skips tenant eligibility page (because they only need to qualify for one type of eligibility)
+
+      expect(page).to have_text I18n.t("state_file.questions.nj_estimated_tax_payments.edit.title", filing_year: filing_year)
+    end
+
+    it "handles property tax both flow - ineligible homeowner and eligible tenant - when low income but meets exemption", required_schema: "nj" do
+
+      advance_to_start_of_intake("Minimal") # low income
+      advance_county_and_municipality
+      advance_disabled_exemption(true) # disabled exemption
+      advance_veterans_exemption
+      advance_medical_expenses
+
+      # household rent/own page
+      choose I18n.t('state_file.questions.nj_household_rent_own.edit.both')
+      click_on I18n.t("general.continue")
+
+      # homeowner eligibility page
+      expect(page).to have_text I18n.t("state_file.questions.nj_homeowner_eligibility.edit.title", filing_year: filing_year)
+      click_on I18n.t("general.continue")
+
+      # homeowner ineligible page
+      property = I18n.t("state_file.questions.nj_ineligible_property_tax.edit.on_home")
+      reason = I18n.t("state_file.questions.nj_ineligible_property_tax.edit.reason_property_taxes")
+      expect(page).to have_text strip_html_tags(I18n.t("state_file.questions.nj_ineligible_property_tax.edit.title_html", property: property))
+      expect(page).to have_text strip_html_tags(I18n.t("state_file.questions.nj_ineligible_property_tax.edit.reason_html", reason: reason))
+      click_on I18n.t("general.continue")
+
+      # tenant eligibility page
+      expect(page).to have_text I18n.t("state_file.questions.nj_tenant_eligibility.edit.title", filing_year: filing_year)
+      check I18n.t('state_file.questions.nj_tenant_eligibility.edit.tenant_home_subject_to_property_taxes')
+      click_on I18n.t("general.continue")
+
+      # skips rent paid page
+
+      expect(page).to have_text I18n.t("state_file.questions.nj_estimated_tax_payments.edit.title", filing_year: filing_year)
+    end
```

Comment:
[boulder] - sand if engineer-only, but if requesting PM review of this too, we should probably DRY these now to be more readable. One way to do it would be anywhere there is currently a comment in this diff could be a helper function instead